Tests/XCUITests/TabsPerformanceTests.swift

```diff
@@ -62,14 +62,17 @@ class TabsPerformanceTest: BaseTestCase {
     func testPerfTabs_3_20tabTray() {
         app.launch()
         waitForTabsButton()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.tabsButton].staticTexts["20"])
+        app.buttons["TabLocationView.reloadButton"].tap()
```

Comment:
Looks like the problem is not with the Tabs button, I tried to reload the page here and this button is not tappable either... See [logs](https://app.bitrise.io/build/2b7d1904-e49d-4052-b894-b6df0f4d2cca?tab=artifacts)
So, I think that the issue is the CFR message that appears in the screenshot of the app at that moment that is focused not allowing the other buttons to be clicked...

![Screenshot_1_0F5B988F-D47F-4D15-A25C-76D70DABAEAC copy](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/02c59db4-4b10-469d-bf86-cd5a5f75d211)